### PR TITLE
Add remoting docs

### PR DESCRIPTION
**Pull request by tylerl0706**
These are some docs to help folks understand how to use VSCode to remote edit/debug scripts!

Hoping this is in the right place.

https://github.com/tylerl0706/vscode-powershell/blob/remoting-doc/docs/remoting.md

### CODE REVIEW

File: docs/remoting.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+![The call to Enter-PSSession](Assets/enterpssession.png)
+
+From there, we can do the exact steps as if we were editing a local script.
+
+1. Run `psedit test.ps1` to open the remote `test.ps1` file
+![psedit the test.ps1 file](Assets/pseditremote.png)
+2. Edit the file/set breakpoints
+![edit and set breakpoints](Assets/addbreakpointremote.png)
+3. Start debugging the remote file
```
Comment (daviwil) [inline docs/remoting.md]: I'd mention the use of F5 again here just in case the skipped the previous section

**Comment by tylerl0706** [comment]
@daviwil I had talked to @SteveL-MSFT about putting some docs here and also posting a blog post - but using the PowerShell docs site might be a good alternative to putting them in the repo.

Either that - or we can throw the images somewhere else (imgur or somewhere) and link them from there though idk if it's ok.

**Comment by daviwil** [comment]
One cheeky thing that I've done in the past is paste the images into a GitHub issue (or PR) and then link to the CDN images that get created...  See CHANGELOG.md for examples of this mischief ;)

File: docs/remoting.md (hunk 1 above)
Comment (tylerl0706) [reply docs/remoting.md]: fixed

**Comment by tylerl0706** [comment]
@daviwil no more images 😄 

Lets call PowerShell Docs out of scope for this PR.

**Comment by tylerl0706** [comment]
I'm going to have to change these when `Open-EditorFile` gets released... you resolve some, you open some I guess 😄

**Comment by tylerl0706** [comment]
FYI, I did a slight update that talks about `Open-EditorFile`.

**Pull request merged**